scripts/templates/selectorsyncset.yaml

```diff
@@ -0,0 +1,16 @@
+
+apiVersion: hive.openshift.io/v1alpha1
+kind: SelectorSyncSet
+metadata:
+  generation: 1
+  name: osd-manged-cluster-config
+  labels:
+    osd-config: True 
```

Comment:
@jewzaam @rafael-azevedo should we make this osdConfig, looks kind of odd but matches convention.

Reply:
This is resolved by the method in the comments below.